Testscripts/Linux/WALA-VERIFY-HOSTNAME-CHANGE.py

```diff
@@ -66,7 +66,7 @@ def ChangeHostName(expectedHost):
     Run("hostname " + changed_hostname)
     fail_error_warn_count = Run("tail -f waagent.log | grep -E 'fail|error|warning' | wc -l")
     RunLog.info("Start to sleep for 120 seconds.")
```

Comment:
log also need change? probably we can have a loop, to check whether the name change within timeout.